docs/authentication.md

```diff
@@ -48,7 +48,7 @@ magic.auth.sign(message)
   });
 });
 
-// supplied seed
+// 2. Supplied seed to derive the keypair from; Buffer or hex encoded string. Seed length can be of any size - although it should still be long and random
```

Comment:
Is N something to determine or you suggest we put a number there?

I'd change _Generate/derive_ to _Generating/deriving_ as the first seems more like something we tell the user to do.

Reply:
N is something to determine, maybe go with 32 bytes here too?

Can also be either `Generating` or `Deriving`, instead of both `Generating/Deriving`. I wasn't sure which one to pick.

Reply:
How about ti following? I removed should because the function works with any length so it's more an advice than a requirement
```suggestion
Generating/deriving a key pair from a supplied seed, and output a Buffer or hex-encoded string. It is advised that the seed has at least 32 characters and is generated by a CSPRNG.
```

Reply:
Looks good!